src/redux/reservationSlice.js

```diff
@@ -0,0 +1,54 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import reservationData from './reservationApi';
+// to come from the api
```

Comment:
Please Kindly add the `reservationData` from the empty `reservationApi` file in order to render the data needed for the apartments API